geolib/models/dstability/internal.py

```diff
@@ -1554,7 +1554,7 @@ class PersistableTangentArea(DStabilityBaseModelStructure):
     Height: Optional[float] = 0.0
     Label: Optional[str] = ""
     Notes: Optional[str] = ""
-    TopZ: Optional[float] = None
+    TopZ: Optional[Union[float, str]] = "NaN"
```

Comment:
In the D-Stability code, Height and TopZ are nullable doubles. So None would be a valid default here

Reply:
To me it makes sense to adjust all occurrences in this module for consistency. Is that OK to you?